README.md

```diff
@@ -649,12 +649,19 @@ You must specify the trailing `--` on the command line to tell runqemu that
 there are no more arguments.  You can then edit the `batch.txt` file to remove
 files, change arguments, etc. and re-run using `--batch-file batch.txt`.
 
-By default, `--make-batch` will sort the list of test files in `alphanum` which
-is US ASCII sort order (using the python `sorted` function).  If you want to use
-a different sort order, use `--make-batch-file-order ORDER`.  Currently, the
-only other `ORDER` is `natural` which is the default filesystem order (whatever
-is the order returned by `glob("tests/tests_*.yml")`).  Note that when you
-specify `--make-batch-file-order`, that implies `--make-batch`.
+By default, `--make-batch` will sort the list of test files in `imagehash`
+order. If you want to use a different order, use `--make-batch-file-order ORDER`.
+Supported `ORDER` values are:
+
+ - `acending`: US ASCII sort order (using Python's `sorted()` function)
```

Comment:
```suggestion
 - `ascending`: US ASCII sort order (using Python's `sorted()` function)
```
found another one - English orthography sucks

Reply:
It does in general, but not in this case -- here it's just my bad typing :grin:  Thanks, fixed.